pilot/pkg/proxy/envoy/v2/discovery.go

```diff
@@ -208,17 +208,17 @@ func (s *DiscoveryServer) ClearCacheFunc() func() {
 			// TODO: metric !!
 			return
 		}
+		s.env.PushContext = push
+		versionLocal := time.Now().Format(time.RFC3339)
 		initContextTime := time.Since(t0)
+		adsLog.Infof("InitContext %v for push takes %s", versionLocal, initContextTime)
```

Comment:
This should be a debug, not info

Reply:
make sense.

Reply:
@hzxuzhonghu I was looking at the surrounding context of this change and it occurred to me that [pilot/pkg/proxy/envoy/v2/discovery.go#L153-L154](https://github.com/hzxuzhonghu/istio/blob/928c5fd038097a7f3f24ca0ad35bb6e8632edcaa/pilot/pkg/proxy/envoy/v2/discovery.go#L153-L154) should do the same thing to avoid logging one version only to push a different version; i.e. assign `versionInfo()` to a local and use that for both logging and push.